package.json

```diff
@@ -35,6 +35,7 @@
     "lodash": "4.6.1",
     "mkdirp": "0.5.1",
     "moment": "2.12.0",
+    "path-is-inside": "1.0.1",
```

Comment:
Haha, that's just the shortcomings of `npm` though. They've retroactively done some damage control by disabling `unpublish` on packages that are >24 hrs old.